main/tests/server/src/com/google/refine/importers/WikitextImporterTests.java

```diff
@@ -51,12 +53,16 @@
 import org.testng.annotations.BeforeTest;
 import org.testng.annotations.Test;
 
-import com.google.refine.importers.WikitextImporter;
+import com.google.refine.model.Recon;
```

Comment:
I have a feeling I may not want to know the answer to this, but why does an importer test have a dependency on reconciliation? That seems very odd.

Reply:
Because the importer itself depends on reconciliation to create reconciled cells?